docs/01_cva6_user/Programmer_View.rst

```diff
@@ -24,28 +24,34 @@ RISC-V specifications allow many variations. This chapter provides more details
 
 RISC-V Extensions
 -----------------
+
+*These are the RISC-V extensions supported by CVA6 and their applicability to configurations:*
+
 .. csv-table::

```

Comment:
It will not be possible to reuse the following table in the Design Documentation. Should we duplicate the information (bad) into different files ?   To be discussed.

Reply:
I tried to solve this in the latest commit. But this implied to have some duplicated information in the same file. Then with templating, you can get rid of information that do not apply to CV32E6?X.